src/ape/types/events.py

```diff
@@ -256,6 +219,16 @@ def abi(self) -> EventABI:
         self._abi = abi
         return abi
 
+    @cached_property
+    def topics(self) -> list[HexStr]:
```

Comment:
If we are worried about name clashing here, we could make this a method, e.g.:

```
def get_topics()`
```

or

```
@property
def encoded_topics
```
or whatever else